component/button/button.html

```diff
@@ -0,0 +1,144 @@
+                        <span class="highlight">btn-outline-secondary</span>and
+                        <span class="highlight">btn-outline-danger</span>
+                    </p>
+
+                    <div class="component-code lg-text">
+                        <div class="container-flex">
+                            <button class="btn btn-outline-primary">Primary</button>
+                            <button class="btn btn-outline-secondary">Secondary</button>
+                            <button class="btn btn-outline-danger">Danger</button>
+                        </div>
+                        <iframe src="https://www.thiscodeworks.com/embed/620277fd234b8a0015700b99" style="width: 100%; height: 125px;" frameborder="0"></iframe>
+                    </div>
+                </article>
+                <article class="comp-desc">
+                    <h2 class="comp-heading">Icon Buttons</h2>
+
+                    <p class="comp-heading-desc lg-text">
+                        Icon buttons allow users to take actions, and make choices, with a single tap.
+                       
+                    </p>
+
+                    <div class="component-code lg-text">
+                        <div class="container-flex">
+                            <button class="btn btn-icon btn-primary ">
+                                <i class="fa fa-shopping-cart"></i>
+                                <span>Add to cart</span>
+                            </button>
+
+                            <button class="btn btn-icon btn-danger">
+                                <i class="fa fa-trash"></i>
+                                <span>Delete</span>
+                            </button>
+                        </div>
+                        <iframe src="https://www.thiscodeworks.com/embed/62027870234b8a0015700b9b" style="width: 100%; height: 190px;" frameborder="0"></iframe>
+                    </div>
+                </article>
+                <article class="comp-desc">
+                    <h2 class="comp-heading">Floating Button</h2>
+
+                    <p class="comp-heading-desc lg-text">
+                        Floating-action button is usually fixed to right-bottom corner.
+                       
+                    </p>
+
+                    <div class="component-code lg-text">
+                        <div class="container-flex">
+                            <div class="floating-btn-box">
+                                <span class="floating-btn x-lg-text">
+                                    <i class="fa fa-plus"></i>
+                                </span>
+                            </div>
+                            <div class="floating-btn-box">
+                                <span class="floating-btn x-lg-text">
+                                    <i class="fa fa-trash"></i>
+                                </span>
+                            </div>
+                    </div>
```

Comment:
Otherwise looks good to me. 

Great work!!

Reply:
Okay. Thanks